esmvaltool/cmorizers/obs/cmorize_obs_fluxcom_gpp_ann.py

```diff
@@ -0,0 +1,86 @@
+"""ESMValTool CMORizer for FLUXCOM GPP data.
+
+Tier
+    Tier 3: restricted dataset.
+
+Source
+    http://www.bgc-jena.mpg.de/geodb/BGI/Home
+
+Last access
+    20190727
+
+Download and processing instructions
+    Download the following files:
```

Comment:
This information is not enough to find the data.
I would add that "FluxCom" has to be selected in both drop-down menus on that page (correct?).
Then a list with two files is shown, which one should be downloaded?

Please document all the required steps in detail.

Reply:
My source files came from JASMIN. For the GPP data the first file listed (the one for Carbon Fluxes is to be downloaded). It in turn has different variables such as GPP and different methods used to produce each variable. This is indicated in the name of the file (variable/method) etc. This I can alter.